package.json

```diff
@@ -155,6 +157,8 @@
     "start-feature": "PORT=3006 OAUTH_ENDPOINT_URL=http://localhost:19090/auth/ API_ENDPOINT_URL=http://localhost:18080/ NN_ENDPOINT_URL=http://localhost:20200/ API_WHEREABOUTS_ENDPOINT_URL=http://localhost:18082/ UPDATE_ATTENDANCE_PRISONS=LEI node backend/index.js",
     "build": "webpack --mode=production",
     "build:analyse": "BUNDLE_ANALYSE=true yarn build",
+    "build-css": "yarn run ./bin/build-css",
+    "css-build": "./bin/build-css",
```

Comment:
are both these required?

Reply:
Unlikely.  I'll check with @Nimphal though when she gets in as to whether there's a method behind the madness.